### DISCUSSION TITLE
personal_sign  Signature TokenPocket

### DISCUSSION THREAD

**Discussion by vsmzz** [WalletConnect/walletconnect-monorepo#4387] [Developer Support] [answered]
Signature TokenPocket Tip: walletconnectv1 is not supported.
But the signature pops up normally after 5-10 seconds.
Is there a way to ignore the TokenPocket prompt: walletconnectv1 is not supported?

**Comment by glitch-txs** [comment]
you need to upgrade to WalletConnect v2 or higher

**Reply by vsmzz** [reply]
I'm using: Kotlin
The introduction of sdk is also the latest:
     implementation(platform("com.walletconnect:android-bom:1.31.0"))
     implementation("com.walletconnect:android-core")
     implementation("com.walletconnect:web3modal")
"Namespace Builder" compliance document incoming
Please tell me how to upgrade and change

**Reply by chendatony31** [reply, marked as answer]
We(TokenPocket) will fix this in the next version, maybe in a week.

**Reply by vsmzz** [reply]
Thank you very much for your reply, which has saved us a lot of trouble
